README.md

```diff
@@ -971,6 +971,46 @@ docker swarm leave --force
 ```
 If the process is successful, a message saying the node has left the swarm will be displayed.
 
+# Image Tag
+
+The `marklogic` image tags allow the user to pin their applications to images for a specific release, a specific minor release, a specific major release, or the latest release of MarkLogic Server
+
+## `{ML release version}-{platform}-{ML Docker release version}`
+
+This tag points to the exact version of MarkLogic Server, the base OS, and the startup scripts version. This allows an application to pin to a very specific version of the image. The image will not be updated without incrementing either the MarkLogic Sever version or the version of the startup scripts.
```

Comment:
I think "startup scripts" is our internal understanding and may not be clear to external audience. Perhaps it should be "Docker scripts" or "supporting scripts"?

Reply:
I think people familiar with Docker images in general understand what a "startup script" is but I don't feel that strongly. Prefer "supporting scripts" if you want to change it.